commitlint.config.cjs

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Can you pull back this update? We'll work on the module/common js transition another time.

Reply:
Done